dashboard/app/models/pd/payment_term.rb

```diff
@@ -62,4 +64,12 @@ def self.for_workshop(workshop)
 
     found_payment_terms[0]
   end
+
+  private
+
+  def sufficient_contract_terms
+    unless per_attendee_payment? || fixed_payment
+      errors.add(:terms, 'Must have either per attendee payment or fixed payment')
```

Comment:
nit: convention here is for the error to be associated with one of the attributes, but there is no attribute called `:terms`. For general errors that refer to the state as a whole, use [:base](http://guides.rubyonrails.org/active_record_validations.html#errors-base).